debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/CoreInfoCommand.java

```diff
@@ -105,7 +105,9 @@ public void run(String command, String[] args, Context context, PrintStream out)
 			/* Print Java Version Info */
 			out.println("JAVA VERSION INFO\n" + properties.get("java.fullversion"));
 			/* Print Java VM Version Info */
-			out.println("JAVA VM VERSION\t- " + properties.get("java.vm.version") + "\n");
+			out.println("JAVA VM VERSION\t- " + properties.get("java.vm.version"));
+			/* Print Java VM Name */
+			out.println("JAVA VM NAME\n" + properties.get("java.vm.name"));
```

Comment:
I suggest this should be presented (on one line) like `JAVA VM VERSION`:
```
out.println("JAVA VM NAME\t- " + properties.get("java.vm.name"));
```

Reply:
Please note the space after `-`.

Reply:
Modified to have it presented on one line

Reply:
I was copying the format of the JAVA VM VERSION as I switched from a newline to a tab for JAVA VM NAME. Will revert back